docs-ref-conceptual/update-azure-cli.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Update the Azure CLI
+description: Reference for the update of Azure CLI
+author: dbradish-microsoft
+ms.author: dbradish
+manager: barbkess
+ms.date: 10/19/2020
+ms.topic: conceptual
+ms.service: azure-cli
+ms.devlang: azurecli 
+ms.custom: devx-track-azurecli
+---
+
+# Update the Azure CLI
+
+You can rely on package managers to update Azure CLI on Windows, macOS and Linux environments (see the `Update` section in each platform-specific install instruction). The CLI also provides in-tool commands to upgrade manually or automatically.
+
+## Manual Update
+[!INCLUDE [az-upgrade](includes/az-upgrade.md)]
+
+`az upgrade` is supported on Windows, macOS and some Linux distros as long as installation is supported. It only supports upgrading to the latest version.
+
+When `azure-cli` is already the latest version, running `az upgrade` will check and update all installed [extensions](azure-cli-extensions-overview.md).
+
```

Comment:
add, "If you are running the Azure CLI through Azure Cloud Shell, you are already using the most recent Azure CLI install."  (or some sentence like this.)

Reply:
changed some wording to clarify for cases when Azure CLI is not the latest on Azure Cloud Shell.